src/main/webapp/dev/js/common/questionRubric.js

```diff
@@ -361,23 +361,13 @@ function highlightRubricCol(index, questionNum, highlight) {
  * @param checkbox the "weights" checkbox
  */
 function moveAssignWeightsCheckbox(checkbox) {
-    const $choicesRow = checkbox.closest('thead').find('tr').eq(0);
-    const $weightsRow = checkbox.closest('thead').find('tr').eq(1);
-    const $choicesRowFirstCell = $choicesRow.find('th').first();
+    const $weightsRow = checkbox.closest('form').find('thead > tr').eq(1);
     const $weightsRowFirstCell = $weightsRow.find('th').first();
 
-    const $checkboxCellContent = checkbox.closest('th').children().detach();
-
-    $choicesRowFirstCell.empty();
-    $weightsRowFirstCell.empty();
-
     if (checkbox.prop('checked')) {
-        $choicesRowFirstCell.append('Choices <span class="glyphicon glyphicon-arrow-right"></span>');
-        $weightsRowFirstCell.append($checkboxCellContent);
-        $weightsRowFirstCell.find('.glyphicon-arrow-right').show();
+        $weightsRowFirstCell.append('Weights <span class="glyphicon glyphicon-arrow-right"></span>');
```

Comment:
Something tells me there will be a minor glitch here. When the user add new question, check `Choices are weighted`, then cancel the form and add it again, the checkbox would still be checked and this would be added again. Can you check if this is the case? If so best we uncheck the checkbox everytime it is shown or cancelled.

Reply:
Well I still think it's more logically sound to reset all of the form's inputs upon cancelling, but that is an issue affecting all question forms and should be addressed in a separate PR altogether.

Reply:
>but that is an issue affecting all question forms.

This was the reason for me to take an alternative approach.

@damithc @xpdavid Should I open a new issue for it, or is there a reason for not resetting the changes upon cancelling?